listenbrainz/model/user.py

```diff
@@ -47,3 +49,6 @@ class UserAdminView(AdminModelView):
         'musicbrainz_id',
         'musicbrainz_row_id',
     ]
+
+    def delete_model(self, model):
+        delete_user(model.musicbrainz_id)
```

Comment:
the `base implementation of `delete_model`](https://flask-admin.readthedocs.io/en/latest/_modules/flask_admin/contrib/sqla/view/#ModelView.delete_model) returns `True` when it succeeds. It's possible that this value is used somewhere else so maybe we should also do it here.
Just checking - I guess that `after_model_delete` isn't useful for us because this is called after the db row itself is deleted, which means that delete_user won't be able to run?

Reply:
We could use `after_model_delete`, I would expect delete_user to continue normally as deleting something that doesn't exist usually behaves. However, it seems counter intuitive to execute the delete twice and I did not want to have different logic to delete a user in different places.